src/pages/contributing/product-development-lifecycle/index.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Product Development Lifecycle
+description:
+  In order to contribute to the Carbon ecosystem, it’s essential that we explain Carbon’s process, define milestones in each phase of contribution, and offer a clear “definition of done” for our contributors. 
+---
+
+<PageDescription>
+
+In order to contribute to the Carbon ecosystem, it’s essential that we explain Carbon’s process, define milestones in each phase of contribution, and offer a clear “definition of done” for our contributors. 
+
+</PageDescription>
+
+<AnchorLinks>
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Discovery</AnchorLink>
+<AnchorLink>Delivery</AnchorLink>
+<AnchorLink>Launch and scale</AnchorLink>
+<AnchorLink>Definition of done</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+IBM teams follow a process framework called the Product Development Lifecycle (PDLC). IBM defines the PDLC as the process of taking a product capability from an idea through its development and release to market, and beyond. Since product development takes place in a continuously evolving market, the framework creates room for validation in every phase of the lifecycle.
```

Comment:
```suggestion
IBM teams follow a process framework called the Product Development Lifecycle (PDLC). IBM defines the PDLC as the process of taking a product capability from an idea through its development and release to market and beyond. Since product development takes place in a continuously evolving market, the framework creates room for validation in every phase of the lifecycle.
```